core/src/main/javascript/reportviewer/reportviewer.js

```diff
@@ -1193,7 +1193,7 @@ define([ 'common-ui/util/util', 'common-ui/util/timeutil', 'common-ui/util/forma
                     hideDlgAndPane(registry.byId('feedbackScreen'));
 
                     //Show loading screen
-                    $('#notification-message').html(_Messages.getString('LoadingPage') + " " + mainJobStatus.page + " " + _Messages.getString('Of') + " " + mainJobStatus.totalPages);
+                    xssUtil.setHtml($('notification-message'), _Messages.getString('LoadingPage') + " " + mainJobStatus.page + " " + _Messages.getString('Of') + " " + mainJobStatus.totalPages);
```

Comment:
I can't find the comment from another PR review where I wrote this, so I'm repeating here.

Note that the query selector passed to jQuery should remain as `#notification-message`.

The XSS Utility `setHtml(elem, html)` method does not support jQuery objects in the first argument, only actual `HTMLElement`.

On the other hand, if we keep using the pattern `$('#notification-message').html(xssUtil.sanitizeHtml(html))` with jQuery instances, likely, the corresponding AppScan findings will not be automatically resolved.

As such, we should consider adding support for jQuery objects to the existing `setHtml(..)` method, and then use it with these jQuery cases.